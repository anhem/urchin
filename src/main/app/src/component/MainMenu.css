#main-menu {
    vertical-align: top;
    text-align: center;
}

#main-menu ul {
    list-style-type: none;
    padding: 0;
}

#main-menu a {
    color: white;
    text-transform: uppercase;
    text-decoration: none;
    font-weight: bold;
}

#main-menu__logo {
    width: 5em;
    height: 5em;
    margin: 0 auto;
}

#main-menu__logo img {
    border-radius: 100%;
    height: 100%;
    width: 100%;
}
