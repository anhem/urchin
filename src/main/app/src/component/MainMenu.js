import React, {Component} from 'react'

class MainMenu extends Component {

    render() {
        return (
            <div id="main-menu">
                MainMenu
                <ul>
                    <a id="main-menu__home" href="/home">Home</a>
                </ul>
                <ul>
                    <a id="main-menu__users" href="/users">Users</a>
                </ul>
                <ul>
                    <a id="main-menu__groups" href="/groups">Groups</a>
                </ul>
                <ul>
                    <a id="main-menu__folders" href="/folders">Folders</a>
                </ul>
            </div>
        )
    }
}

export default MainMenu