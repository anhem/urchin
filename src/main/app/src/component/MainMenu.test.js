import React from 'react';
import {mount} from 'enzyme'
import MainMenu from "./MainMenu";

describe('MainMenu', () => {

    it('renders without crashing', () => {
        expect(mount(<MainMenu/>).length).toEqual(1);
    });
});
