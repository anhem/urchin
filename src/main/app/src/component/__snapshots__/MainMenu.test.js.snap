// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainMenu match snapshot 1`] = `
<div
  id="main-menu"
>
  MainMenu
  <ul>
    <a
      href="/home"
      id="main-menu__home"
    >
      Home
    </a>
  </ul>
  <ul>
    <a
      href="/users"
      id="main-menu__users"
    >
      Users
    </a>
  </ul>
  <ul>
    <a
      href="/groups"
      id="main-menu__groups"
    >
      Groups
    </a>
  </ul>
  <ul>
    <a
      href="/folders"
      id="main-menu__folders"
    >
      Folders
    </a>
  </ul>
</div>
`;
